### PR TITLE
REM-634

### PR DESCRIPTION
**Pull request by imtithalhalalo**
To test try on page : 'au-locked-records'
custom date time picker
to test in page 'au-locked-records' check the last field

### CODE REVIEW

**Comment by Ghinwa-Arab** [comment]
image

 last two fields have same labels

File: src/components/Inputs/CustomDateTimePicker.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+// ** React Imports
```
Comment (Ghinwa-Arab) [inline src/components/Inputs/CustomDateTimePicker.js:1]: remove

File: src/components/Inputs/CustomDateTimePicker.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+import { useRef, useState } from 'react'
+
+// ** MUI Imports
```
Comment (Ghinwa-Arab) [inline src/components/Inputs/CustomDateTimePicker.js:4]: remove

File: src/components/Inputs/CustomDateTimePicker.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
+// ** React Imports
+import { useRef, useState } from 'react'
```
Comment (Ghinwa-Arab) [inline src/components/Inputs/CustomDateTimePicker.js:2]: remove useRef

File: src/components/Inputs/CustomDateTimePicker.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+import PopperComponent from '../Shared/Popper/PopperComponent'
+import { DateTimePicker } from '@mui/x-date-pickers'
+
+const CustomDateTimePicker = ({
+  name,
+  label,
+  value,
+  onChange,
+  error,
+  helperText,
```
Comment (Ghinwa-Arab) [inline src/components/Inputs/CustomDateTimePicker.js:23]: do we need helperText?

**Comment by NoraBitarSM** [comment]
the dropdown is so big, can we decrease the size a little bit ? 
image

**Review by Alinadimmokdad — APPROVED** [review]

**Review by RebeccaFrem02 — APPROVED** [review]

**Pull request merged**